python/tests/unit/arcticdb/version_store/test_filtering.py

```diff
@@ -834,15 +836,34 @@ def test_filter_numeric_isin_unsigned(lmdb_version_store):
 @settings(deadline=None)
 @given(
     df=dataframes_with_names_and_dtypes(["a"], integral_type_strategies()),
-    vals=st.frozensets(integral_type_strategies(), min_size=1),
+    vals=st.frozensets(unsigned_integral_type_strategies(), min_size=1),
 )
-def test_filter_numeric_isnotin(lmdb_version_store, df, vals):
+def test_filter_numeric_isnotin_unsigned(lmdb_version_store, df, vals):
     numeric_isin_asumptions(df, vals)
     q = QueryBuilder()
     q = q[q["a"].isnotin(vals)]
     pandas_query = "a not in {}".format(list(vals))
     generic_filter_test(lmdb_version_store, "test_filter_numeric_isnotin", df, q, pandas_query)
 
+@use_of_function_scoped_fixtures_in_hypothesis_checked
+@settings(deadline=None)
+@given(
+    df=dataframes_with_names_and_dtypes(["a"], integral_type_strategies()),
+    vals=st.frozensets(signed_integral_type_strategies(), min_size=1),
+)
+def test_filter_numeric_isnotin_signed(lmdb_version_store, df, vals):
+    numeric_isin_asumptions(df, vals)
+    q = QueryBuilder()
+    q = q[q["a"].isnotin(vals)]
+    pandas_query = "a not in {}".format(list(vals))
+    generic_filter_test(lmdb_version_store, "test_filter_numeric_isnotin", df, q, pandas_query)
```

Comment:
The test bodies look identical to me. What's the intention of having two tests?

Reply:
it is something I discussed with Matt, in order to separate the signed and unsigned cases in two tests explicitly, since we are testing separate things.

Reply:
I did say to create two tests in fairness. A four line duplicate in two tests isn't too bad imo.